src/components/DeclareLostDialog/DeclareLostDialog.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Modal } from '@folio/stripes/components';
+import { stripesShape } from '@folio/stripes/core';
+
+import DeclareLostInfo from './DeclareLostInfo';
+
+class DeclareLostDialog extends React.Component {
+  static propTypes = {
+    stripes: stripesShape.isRequired,
+    onClose: PropTypes.func.isRequired,
+    open: PropTypes.bool.isRequired,
+    loan: PropTypes.object.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.connectedDeclareLostDialogInfo = props.stripes.connect(DeclareLostInfo);
```

Comment:
@YevheniiMaltsev  could we use `stripesConnect` from `import { stripesConnect } from '@folio/stripes/core';` in `DeclareLostInfo`? This way you could avoid creating a constructor and perhaps even turn it into a functional component.